### PR TITLE
Issue 30

### LINKED ISSUE #30: Add events info page
dod: successfully adding an event and having it show on the events page. have the events auto-populate to ensure future expandability.

### LINKED ISSUE #81: adding live data to events page
DOD: display event data from room and display the data when navigating to info page

### PR DESCRIPTION
**Pull request by haythamodeh**

### CODE REVIEW

**Review by isaakwheeler — APPROVED** [review]

File: app/src/main/java/edu/cascadia/mobas/campusguidebook/ui/events/EventsFragment.java [hunk 1]
```diff
@@ -34,11 +35,17 @@ public View onCreateView(@NonNull LayoutInflater inflater,
...
             Navigation.findNavController(card1).navigate(R.id.fragment_Event_Info);
         });
 
+        addEventBtn.setOnClickListener(view -> {
+            Navigation.findNavController(addEventBtn).navigate(R.id.addEventFragment);
+        });
+
         //final TextView textView = binding.spinner;
         //slideshowViewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
         //    @Override
```
Comment (gomezfox) [inline app/src/main/java/edu/cascadia/mobas/campusguidebook/ui/events/EventsFragment.java]: Should delete commented out code that is no longer useful

File: app/src/main/java/edu/cascadia/mobas/campusguidebook/data/repository/AppRepository.java [hunk 2]
```diff
@@ -59,6 +63,17 @@ public static AppRepository getInstance(AppDatabase db, AppExecutors executors)
         return mAppDatabase.SustainabilityDao().getAllSustainability();
     }
 
+    public boolean addNewEvent(String eventName, String eventDescription, String eventLocation, ZonedDateTime eventDateTime) {
+        Event event = new Event(eventName, eventDescription, eventLocation, eventDateTime);
+        try {
+            this.insert(event);
+            return true;
+        } catch (Exception e) {
```
Comment (gomezfox) [inline app/src/main/java/edu/cascadia/mobas/campusguidebook/data/repository/AppRepository.java:74]: Should wrap this in an executor. Insert occurs on another thread, so true will be returned before any results are known.

**Review by gomezfox — APPROVED** [review]
A couple problems that can be dealt with in the future, but overall, good work.

**Review by odjuliarso — APPROVED** [review]

**Comment by Bansenauer-Cascadia** [comment]
@haythamodeh - Your branch is not current with develop. Be sure to pull the current develop branch to your computer, then merge develop INTO your issue_30 branch. Resolve the conflicts, commit your branch and push it back up to GitHub. This is the GitFlow method we're using.

NOTE: your pull request will need to go through the approval process again. That is normal.

**Pull request merged**